src/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![forbid(unsafe_code)]
 #![deny(unused)]
 #![deny(dead_code)]
+#![warn(missing_debug_implementations)]
```

Comment:
I am a bit undecided. I don't think we should be adding `Debug` implementations to types by default as it slows compile time and bloats the compiled binary.

Do you feel strongly about this? What do you think of removing this line but keeping the new `derive` statements below?

```suggestion
```

Reply:
This only lints for public types I believe. So it's not every type.

But I don't feel _super_ strongly about it. Though I think it is a good idea.

Your call :)

Reply:
> This only lints for public types I believe. So it's not every type.

Good point. Thanks. Let's keep it as is then.